src/charactersheet/viewmodels/common/wizard/index.html

```diff
@@ -645,6 +645,62 @@ <h3>Tell Us About Your Campaign</h3>
         >
       </div>
     </div>
+
+    <hr />
+
+    <div class="form-group">
+      <label for="playerName" class="col-sm-2 control-label">
+        Setting
+      </label>
+      <div class="col-sm-10">
+        <input
+          type="text"
+          class="form-control"
+          id="motif"
+          maxlength="128"
+          placeholder="Where does this adventure take place?"
+          data-bind="
+            textInput: setting,
+            autocomplete: { source: settingOptions, onselect: setting }
+          "
+        >
+      </div>
+    </div>
+    <div class="form-group">
+      <label for="playerName" class="col-sm-2 control-label">
+        Motif
+      </label>
+      <div class="col-sm-10">
+        <input
+          type="text"
+          class="form-control"
+          id="motif"
+          maxlength="128"
+          placeholder="Noir/Mystery/Adventure"
```

Comment:
The other placeholders are asking a question. Maybe this one could also ask a question like `What is the general atmosphere of the campaign?`